src/common/options.cc

```diff
@@ -785,6 +785,10 @@ std::vector<Option> get_global_options() {
     .set_default(5)
     .set_description("Zlib compression level to use"),
 
+    Option("compressor_zstd_level", Option::TYPE_INT, Option::LEVEL_ADVANCED)
+    .set_default(1)
```

Comment:
better off changing the default setting from 5 to 1 in a separated commit.

Reply:
I've created this ticket & PR for switching the default:
https://tracker.ceph.com/issues/44724
https://github.com/ceph/ceph/pull/34133

Reply:
thanks Bryan! we can have #34133 in this PR. the whole idea of having it in a separated commit is to split the changes which change the behavior from those don't. and to have different PR is mainly to reduce the efforts of PRs are touching different area of Ceph and have different level of risk and schedule.

but per your description, this changes are of low risk, and both of them are changing the same area and can be exercised by the same set of tests.

Reply:
I understand.  I've closed that PR in favor of separate commits in this PR.

Reply:
thanks!